pegasus/sites.v3/code.org/views/theme_google_analytics.haml

```diff
@@ -1,34 +1,38 @@
 :javascript
-  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-  ga('create', 'UA-37745279-1', '#{request.site}');
+  (function () {
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+    ga('create', 'UA-37745279-1', '#{request.site}');
 
-  ga('require', 'linkid', 'linkid.js');
+    ga('require', 'linkid', 'linkid.js');
 
-  var currentLocale = readCookie("language_");
-  if (currentLocale && currentLocale != "")
-  {
-    var currentLanguage = currentLocale.substring(0,2);
-    if (currentLanguage && currentLanguage != "")
+    var currentLocale = readCookie("language_");
+    if (currentLocale && currentLocale != "")
     {
-      ga('set', 'dimension1', currentLanguage);
+      var currentLanguage = currentLocale.substring(0,2);
+      if (currentLanguage && currentLanguage != "")
+      {
+        ga('set', 'dimension1', currentLanguage);
+      }
     }
-  }
 
-  ga('send', 'pageview');
-  function trackEvent(category, action, label, value) {
-    ga('send', 'event', category, action, label, value);
-  }
+    ga('send', 'pageview');
+    function trackEvent(category, action, label, value) {
+      ga('send', 'event', category, action, label, value);
+    }
 
-  function readCookie(name) {
-    var nameEQ = name + "=";
-    var ca = document.cookie.split(';');
-    for(var i=0;i < ca.length;i++) {
-      var c = ca[i];
-      while (c.charAt(0)==' ') c = c.substring(1,c.length);
-      if (c.indexOf(nameEQ) == 0) return c.substring(nameEQ.length,c.length);
+    function readCookie(name) {
+      var nameEQ = name + "=";
+      var ca = document.cookie.split(';');
+      for(var i=0;i < ca.length;i++) {
+        var c = ca[i];
+        while (c.charAt(0)==' ') c = c.substring(1,c.length);
+        if (c.indexOf(nameEQ) == 0) return c.substring(nameEQ.length,c.length);
+      }
+      return null;
     }
-    return null;
-  }
+    window.trackEvent = trackEvent;
+    window.readCookie = readCookie;
```

Comment:
not clear to me why `readCookie` is in a file about google analytics, but i didn't feel like moving this right now